Resources/Prototypes/_NF/Catalog/Fills/Backpacks/npc_loot_mercenaries.yml

```
@@ -472,6 +474,8 @@
     # Weapons
     - id: WeaponSniperRepeaterExpedition
       prob: 0.9
+    - id: WeaponRifleMR8 # mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)